.travis.yml

```diff
@@ -13,4 +13,11 @@ before_cache:
 cache:
   directories:
     - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
\ No newline at end of file
+    - $HOME/.gradle/wrapper/
+
+
+notifications:
+  slack:
+    on_success: always
+    on_failure: change
+    secure: WOingZJ/nk8Mc5JXbHgE5Ux1YwpsrWryft5TH8kPBaIqTmX6qlcu3KSdLktwgiPgXJiOWpR6UitX9+xfalcApEfS77ZNQeUFafDYHv17nS6IAHe/A4vOAIElXfpS2KnY0aqxwWhgKCsehio49SPY7WHq9PtnvauEdkX8bca5FHk=
```

Comment:
That makes sense! I was just concerned it was a secret key. I didn't know
Travis supported encrypted tokens!
On Thu, May 12, 2016 at 2:00 PM Peter Mitrano <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/wpilibsuite/allwpilib/pull/11#discussion_r63067349>:
>
> > \ No newline at end of file
> > +    - $HOME/.gradle/wrapper/
> > +
> > +
> > +notifications:
> > +  slack:
> > +    on_success: always
> > +    on_failure: change
> > +    secure: WOingZJ/nk8Mc5JXbHgE5Ux1YwpsrWryft5TH8kPBaIqTmX6qlcu3KSdLktwgiPgXJiOWpR6UitX9+xfalcApEfS77ZNQeUFafDYHv17nS6IAHe/A4vOAIElXfpS2KnY0aqxwWhgKCsehio49SPY7WHq9PtnvauEdkX8bca5FHk=
>
> I'm with jonathan on this one.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/wpilibsuite/allwpilib/pull/11/files/c5f84d25c7da01f01ee406072fac0766b557e404#r63067349>
>